spec/models/interest_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe Interest do
+  it 'can have type only from predefined category: health, hobby, work' do
```

Comment:
Well, that spec checks if it can have health, hobby and work types, but it does not check if that's the **only** possibility. Given that, that spec seems to have quite a low value for me.